frontend/src/pages/landing/landing.tsx

```diff
@@ -0,0 +1,35 @@
+import { type FC } from 'react';
+
+import { Link } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/app-route.enum';
+import { useEffect, useMousePosition, useState } from '~/libs/hooks/hooks.js';
+
+import styles from './landing.module.scss';
+
+const DEEP = 0.05;
+
+const Landing: FC = () => {
+  const { x, y } = useMousePosition();
+  const [backgroundPosition, setBackgroundPosition] = useState('center');
+
+  useEffect(() => {
+    const offsetX = (x - window.innerWidth / 2) * DEEP;
```

Comment:
Could be moved to helper
```suggestion
    const offsetX = getAxisOffset({ axisValue, innerAxisValue });
    
    /** helper fn */
    const getAxisOffset = ({axisValue, innerAxisValue }) => {
      return (axisValue - innerAxisValue / 2) * DEEP;
    };
```

Reply:
You gave an example of a helper that accepts an object with two values, but in the function I will also use DEEP, which can also be changed, for more detailed configuration. Does it make sense to pass it separately to the function? I would.

Reply:
You can make it as optional prop to arguments with default value

Reply:
moved to helper - [536ac50](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/26/commits/536ac506aa50ee35f76786ecf04b913cf8e95462)